app/controllers/marks_controller.rb

```diff
@@ -0,0 +1,104 @@
+class MarksController < ApplicationController
+  before_action :results, only: [:vimeo_subscribe]
+  before_action :require_spy, only: [:index, :results, :show]
+
+  def twitter
+    Seemore::Application.config.twitter
```

Comment:
Having this helper method is great, much cleaner to read than calling the long chain ```Seemore::Application.config.twitter``` in another method. If it's used by other controllers, consider moving it to the ApplicationController. If it's just for this controller, keep it here and move it to ```private```.